test/performance/odp_ipsec.c

```diff
@@ -1008,6 +1086,11 @@ static void parse_args(int argc, char *argv[], ipsec_args_t *cargs)
 		}
 	}
 
+	if (cargs->in_flight < cargs->burst_size) {
+		printf("-f (flight) should be greater than -b (burst)\n");
```

Comment:
Burst size should not exceed the maximum number of in-flight packets, but it may equal it.

Reply:
@JannePeltonen  Shall I update the text as,

`printf("-f (flight) must be greater than or equal to -b (burst)\n");`

Or did you mean use the above text verbatim?

Reply:
Actually, your proposed update looks ok whereas copying my comment verbatim would make the error message more complicated than it needs to be, I think.